### DISCUSSION TITLE
What country is the main producer of vanilla and is also the 4th largest island in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#243] [Galaxy Trivia] [answered]
**Hint:** 🇲🇬

**Comment by thuongtruong1009** [comment, marked as answer]
Madagascar is the main producer of vanilla and is also the 4th largest island in the world. The island is located off the southeast coast of Africa in the Indian Ocean. Madagascar is known for its unique biodiversity and is home to many species of plants and animals that cannot be found anywhere else in the world. Vanilla is one of the most important agricultural products of Madagascar, accounting for a significant portion of the country's export earnings.
